File: backend/BusinessService/Common/Bases/Response/CmDownloadResponseBase.cs
﻿// *******************************************************************
// 業務名称　: 健康管理システム
// 機能概要　: ダウンロード処理
// 　　　　　　レスポンスインターフェースベース
// 作成日　　: 2023.02.14
// 作成者　　: 蔣
// 変更履歴　:
// *******************************************************************
namespace BCC.Affect.Service
{
    public class CmDownloadResponseBase : CmPreviewResponseBase
    {
        public EnumDownloadType downloadtype { get; set; } = EnumDownloadType.Data; //ファイルタイプ
        public string filefullnm { get; set; }                                      //ファイル名(パス含む)
    }
}

File: GJSBackend/BusinessService/Common/Bases/Response/CmDownloadResponseBase.vb
﻿' *******************************************************************
' 業務名称　: 互助防疫システム
' 機能概要　: ダウンロード処理
' 　　　　　　レスポンスインターフェースベース
' 作成日　　: 2024.07.14
' 作成者　　: 蔣
' 変更履歴　:
' *******************************************************************
Namespace Jbd.Gjs.Service
    Public Class CmDownloadResponseBase
        Inherits CmPreviewResponseBase
        Public Property downloadtype As EnumDownloadType = EnumDownloadType.Data 'ファイルタイプ
        Public Property filefullnm As String                                      'ファイル名(パス含む)
    End Class
End Namespace
